aptos-move/move-examples/mint_nft_v2/1-Create-NFT/sources/create_nft.move

```diff
@@ -0,0 +1,187 @@
+/// This is a tutorial intended to mirror the `mint_nft` example in move-examples as closely as possible, with the only difference
+/// being that we're using the no code token from aptos_token to demonstrate how to do the tutorial with token v2 instead of token v1.
+///
```

Comment:
We should also include some brief explanations of the core parts of the Move code:
1. Ticket model (tickets are used to mint real tokens later on?)
2. Collection creation
3. Token minting
4. Property Metadata